techniques/system/common/1.0/cf-serverd.st

```diff
@@ -79,7 +79,7 @@ bundle common def
 
    # List here the IP masks that we grant access to on the server
 
-    policy_server::
+    policy_server.(agent|server)::
```

Comment:
it doesn't need to be validated during cf-promises. it may help on large networks